CMakeLists.txt

```diff
@@ -167,10 +167,21 @@ enable_testing()
 # Custom target for building tests. all excludes tests by default
 add_custom_target(all-tests)
 
+# Custom install target for docs to depend on.
+add_custom_target(
+  install-scipp COMMAND cmake --build ${CMAKE_CURRENT_BINARY_DIR} --target
+                        install
```

Comment:
Does this fix #1655?

Reply:
Depends. It respects the `-j` option when building scipp but not with the documentation itself.

Reply:
Great, I think I'd call that fixed then.